HTML/mockup-axit/js/content.js

```diff
@@ -0,0 +1,13 @@
+export const paragraph = [
```

Comment:
we have objects in js so why not use one structure like tab = { title: 'Title', description: 'sometext'}